alarms/FacebookPages/facebookpages_alarm.py

```diff
@@ -56,20 +56,16 @@ def connect(self):
 	def set_alert(self, settings, default):
 		alert = {}
 		alert['message'] = settings.get('message', default['message'])
-		
-		if parse_boolean(settings.get('location', default['location'])):
-			alert['attachment'] = { 'link': "<gmaps>" }
+		alert['attachment'] = { 'link': settings.get('link', default['link']) }
```

Comment:
Is there a reason why you left this as a dict? I think that just making it `alert['link'] = settings.get('link', default['link']) would be more inline with the rest of Alarms.